<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>customAddOperationDialog</class>
 <widget class="QDialog" name="customAddOperationDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>348</width>
    <height>207</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>CustomYesNoDialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
	font: 600 9pt &quot;Segoe UI&quot;;
	border: none;
	background-color: #dfdfdf;
	padding: 0;
	margin: 0;
	color: #324b4c;
}

QDialog{
	background-color: rgba(255, 255, 255, 0);

}

QDateEdit::up-button, QDateEdit::down-button{
	width: 0px;
}


QLineEdit, QDateEdit{
	border: none;
	font-size: 9pt;
	border-bottom: 1px solid #7c9399;
}

QComboBox{
	background-color: #dfdfdf;
	border-bottom: 1px solid #7c9399;
	border-radius: 3px;
}

QComboBox::drop-down{
	image: url(':/icons/app/assets/icons/Alt-Arrow-Down--Streamline-Solar-Broken.svg');
	width: 16px;
	height: 16px;
}

QComboBox:hover{
	color: #2b2b2c;
	font: 600 9pt &quot;Segoe UI&quot;;
	border-radius: 0px;
	alternate-background-color: #2c313c;
}
QComboBox::item:selected{
	color: #2b2b2c;
	font: 700 9pt &quot;Segoe UI&quot;;
	border-radius: 0px;
}
QComboBox::item{
	color: #324b4c;
	font: 600 9pt &quot;Segoe UI&quot;;
	height: 10px;  
	border: 4px solid transparent;
	padding: 0px 8px 0px 8px;
}

#calStartBtn, #calLeavingBtn {
	background-color: #dfdfdf;
	padding: 0px;
}

#mainWidget{
	border-radius: 15px;
}

#widget * {
	font-size: 9pt;
}

#dialogTitle {
	font-size: 10pt;
	font-weight: 700;
}
#yesBtn {
	padding: 0px;
}

#yesBtn:hover{
	icon: url(:/icons/app/assets/icons/Check-Square--Streamline-Solar-Broken-#008b69.svg);
	color: #008b69;
	padding: 0px;
}


QScrollBar:vertical {
	width: 20px;
	background: none;
}

QScrollBar::add-line:vertical, QScrollBar::sub-line:vertical {
	background: #dfdfdf;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
    <widget class="QWidget" name="mainWidget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>10</number>
      </property>
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="topMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <property name="bottomMargin">
       <number>10</number>
      </property>
      <item alignment="Qt::AlignHCenter">
       <widget class="QWidget" name="widget_2" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>5</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>10</number>
         </property>
         <item>
          <widget class="QLabel" name="dialogTitle">
           <property name="text">
            <string>Добавяне на операция</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <layout class="QFormLayout" name="addOperationLayout">
           <property name="horizontalSpacing">
            <number>15</number>
           </property>
           <property name="verticalSpacing">
            <number>10</number>
           </property>
           <item row="0" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Име операция</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Длъжност</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="operationNameLineEdit">
             <property name="minimumSize">
              <size>
               <width>180</width>
               <height>0</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QComboBox" name="operTypeComboBox"/>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
       <widget class="QWidget" name="buttonsWidget" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>Segoe UI</family>
          <pointsize>9</pointsize>
          <italic>false</italic>
          <bold>true</bold>
         </font>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>10</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="yesBtn">
           <property name="font">
            <font>
             <family>Segoe UI</family>
             <pointsize>9</pointsize>
             <italic>false</italic>
             <bold>true</bold>
            </font>
           </property>
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="text">
            <string> ДА</string>
           </property>
           <property name="icon">
            <iconset resource="../../../resources.qrc">
             <normaloff>:/icons/app/assets/icons/Check-Square--Streamline-Solar-Broken.svg</normaloff>:/icons/app/assets/icons/Check-Square--Streamline-Solar-Broken.svg</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>22</width>
             <height>22</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../resources.qrc"/>
 </resources>
 <connections/>
</ui>
